1-JavaScript/Week3/homework/ex4-shoppingCart.js

```diff
@@ -21,8 +21,14 @@ you have more than 3 items in your shopping cart the first item gets taken out.
 -----------------------------------------------------------------------------*/
 const shoppingCart = ['bananas', 'milk'];
 
-function addToShoppingCart(/* parameters go here */) {
+function addToShoppingCart(/* parameters go here */ item) {
   // TODO complete this function
+
+  shoppingCart.push(item);
+  if (shoppingCart.length > 3) {
+    shoppingCart.shift();
+  }
+  console.log('You bought ' + shoppingCart.join(', ') + '!');
 }
 
 addToShoppingCart('chocolate'); // Returns "You bought bananas, milk, chocolate!"
```

Comment:
Notice that the objective is for the function to _**return**_ a value. Currently your function _**console.logss**_ the message. It is the correct message, but you should return it, instead of logging it.